docs/guides/contribute/rockydocs_formatting.md

```diff
@@ -291,6 +291,57 @@ And here's how that renders when displayed:
 
 A workstation type installation (with graphical interface) starts this interface on terminal 1. Linux being multi-user, it is possible to connect several users several times, on different **physical terminals** (TTY) or **virtual terminals** (PTS). Virtual terminals are available within a graphical environment. A user switches from one physical terminal to another using <kbd>Alt</kbd> + <kbd>Fx</kbd> from the command line or using <kbd>CTRL</kbd> + <kbd>Alt</kbd> + <kbd>Fx</kbd>.
 
+## Superscript, Subscript and Special Symbols
+
+Superscript and subscript notation are not normal markdown, but are supported in Rocky Linux documentation via the HTML tags used for the same purpose. Superscript places text entered between the tags slightly above the normal text, while subscript places the text slightly below. Superscript is by far the most commonly used of these two in writing. Some special characters already appear in superscript without adding the tags, but you can also combine the tag to change the orientation of those characters as seen with the copyright symbol below. You can use superscript to:
+
+* represent ordinal numbers, such as 1<sup>st</sup>, 2<sup>nd</sup>, 3<sup>rd</sup>
+* copyright and trademark symbols, like <sup>&copy;</sup>, <sup>TM</sup> or &trade;, &reg;
+* as notation for references, such as this<sup>1</sup>, this<sup>2</sup> and this<sup>3</sup>
```

Comment:
add comma after second item in list.

Reply:
Technically, it's the same item. TM or $trade; They both stand for a trademark but the second renders more closely to the actual trademark symbol. That said, I'll add a comma. ;-)

Reply:
Thank you for your review!!!